crds/v1/fission.io_httptriggers.yaml

```diff
@@ -79,12 +79,14 @@ spec:
               method:
                 description: HTTP method to access a function.
                 type: string
+              prefix:
+                description: 'Prefix with which functions are exposed. NOTE: Prefix takes precedence over URL/RelativeURL. Prefix takes precendence over RelativeURL Note that it does not treat slashes specially ("/foobar/" will be matched by the prefix "/foo") so you may want to use a trailing slash here.'
+                type: string
               relativeurl:
                 description: RelativeURL is the exposed URL for external client to access a function with.
                 type: string
             required:
             - functionref
-            - relativeurl
```

Comment:
I don't know if there is a way in CRD validation to say that one of the `relativeURL` or `prefix` is needed?